test/endtoend/utils.js

```diff
@@ -93,8 +106,72 @@ class CompanionService {
   }
 }
 
+const express = require('express')
+class StaticServerService {
+  constructor ({ folders, staticServerPort = 4567 }) {
+    this.folders = folders
+    this.port = staticServerPort
+  }
+
+  async onPrepare () {
+    if (!this.folders) return
+
+    this.app = express()
+
+    for (const desc of this.folders) {
+      this.app.use(desc.mount, express.static(desc.path))
+    }
+
+    const listen = promisify(this.app.listen.bind(this.app))
+
+    this.server = await listen(this.port)
+  }
+
+  async onComplete () {
+    if (this.server) {
+      const close = promisify(this.server.close.bind(this.server))
+      await close()
+    }
+    this.app = null
+  }
+}
+
+const tus = require('tus-node-server')
```

Comment:
`master.tus.io` is tusd and that is the golden up-to-date tus integration, so we are loosing a bit of that here. `tus-node-server` is outdated, according to Marius. But I see the benefit of not relying on `master.tus.io` of course.

Reply:
Yeah, I wanted to use tusd first, but that was a little harder to get going on CI. I think having tus-node-server on CI is in general good because it's better for Uppy if we can also upload to less-than-perfect tus servers, but ideally we'd have both…